index.js

```diff
@@ -46,6 +48,7 @@ module.exports = {
         'if (typeof Bugsnag !== "undefined") {',
         'Bugsnag.releaseStage = "' + releaseStage + '";',
         'Bugsnag.notifyReleaseStages = ["' + envArray.join('","') + '"];',
+        bugsnagConfig.endpoint ? 'Bugsnag.endpoint = ["' + bugsnagConfig.endpoint + '"];' : '',
```

Comment:
Happy to merge this for now but we should tidy up the way this code gets constructed.

It's pretty messy as it stands.